carpool/forms.py

```diff
@@ -25,7 +24,7 @@ class Meta:
 class PoolForm(forms.ModelForm):
     user = forms.ModelChoiceField(queryset=User.objects.all(), required=False)
     tot = forms.IntegerField(widget=forms.NumberInput(), label="Seats")
-    dateTime = forms.DateTimeField(widget=forms.widgets.DateTimeInput(), label="Date Time YYYY-MM-DD HH:MM")
+    dateTime = forms.CharField(widget=forms.TextInput(attrs={'id':'datepicker-3'}))
```

Comment:
This was supposed to input both the date and time of the ride.

Reply:
yes it is

Reply:
the css and js for this id is different in add.html

Reply:
Oh ok.